### ISSUE TITLE
✨  light control

### ISSUE THREAD

**Issue by tzfx** [tzfx/owrn#5]
Add light on/off support

It appears that separate front/rear light control is supported in BT characteristics. 
Allow both/either to be toggled.

**Comment by MitchellHayes** [comment]
https://youtu.be/-GxnO1z7PE4?t=1906 You can control top LEDs as forward and rear LEDS; all of these should be fully addressable. They even talk about a patch that lets you tie duty cycle to the top LED.

**Comment by tzfx** [comment]
👀 

```js
import { useBLECharacteristic } from "./useBLECharacteristic"

export const useLights = () => {
  const { lights, writeLights } = useBLECharacteristic("lights", {
    read: true,
    write: true,
    notify: true,
  })

  const setLights = async (enabled, brightness = 0) => {
    console.log("setting", enabled, brightness)
    let buffer, view
    brightness = Math.round(brightness * 255)
    buffer = new Uint8Array(2)
    view = new DataView(buffer.buffer)
    view.setUint8(1, enabled)
    view.setUint8(0, brightness)
    writeLights(view)
  }

  const parseLights = (view) => {
    if (!view) return null
    return { enabled: view.getUint8(1) === 1, brightness: view.getUint8(0) }
  }

  return {
    lights: parseLights(lights),
    setLights,
  }
}

```

**Comment by tzfx** [comment]
Closed by #27 -- I think brightness control may be a lie. I could not get it working on my pint-- it appears to always read 0 no matter what I wrote to the characteristic.

Even at first read-- when lights come back as "1", brightness was still 0.
